setup.cfg

```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 88
+ignore =  N806,E226,W504
```

Comment:
I'd prefer not to have these warnings commented out globally. I know you want to keep the capital `T`, but my guess is that having flake8 check more things is better than being able to use `T`. However, I am not going to push you here, just stating my point of view.